citrination_client/search/core/query/data_query.py

```diff
@@ -0,0 +1,91 @@
+from pypif.util.serializable import Serializable
+
+from citrination_client.search.dataset.query.dataset_query import DatasetQuery
+from citrination_client.search.pif.query.pif_system_query import PifSystemQuery
+
+
+class DataQuery(Serializable):
+    """
+    Query against dataset metadata, PIF content, file content, or some combination of those types.
+    """
+
+    def __init__(self, logic=None, simple=None, dataset=None, system=None, query=None, **kwargs):
+        """
+        Constructor.
+
+        :param logic: The logic to apply to the query ('SHOULD', 'MUST', 'MUST_NOT', or 'OPTIONAL').
+        :param simple: String with the simple search to run against all fields.
```

Comment:
Could you add a test for simple queries?

Reply:
bump

Reply:
@maxhutch added